x-pack/plugins/infra/server/graphql/nodes/schema.gql.ts

```diff
@@ -14,6 +14,7 @@ export const nodesSchema: any = gql`
 
   type InfraNodePath {
     value: String!
+    displayName: String
```

Comment:
Being able to rely on non-`null` here might simplify handling of the api response. We could always fall back to using the `value` (or rather `id` as suggested above) for the name as well. Or do you envision a situation where signaling the lack of a name would be important?